HKMP/Game/Client/Entity/MantisLord.cs

```diff
@@ -0,0 +1,907 @@
+using System.Collections.Generic;
+using Hkmp.Fsm;
+using Hkmp.Networking.Client;
+using Hkmp.Util;
+using HutongGames.PlayMaker.Actions;
+using UnityEngine;
+
+namespace Hkmp.Game.Client.Entity {
+
+    public class MantisLordS1 : MantisLordSub {
```

Comment:
By making this class `sealed` it will ensure that the virtual method call in the constructor never causes unexpected behaviour. I know that we will never inherit this class, but my IDE complains about the eventuality.
```suggestion
    public sealed class MantisLordS1 : MantisLordSub {
```